js/lib/menuUtil.js

```diff
@@ -0,0 +1,166 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public * License, v. 2.0. If a copy of the MPL was not distributed with this file,
```

Comment:
new files should use the new directory structure - see https://github.com/brave/browser-laptop/blob/master/docs/directoryStructure.md